Fix a watcher task name

alexandair opened the pull request:
Change a watcher task name from "Watch new files" to "WatchMyFolder" to be in sync with images.

azurecla commented:
Hi __@raymondlaghaeian__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

alexandair committed:
Fix a watcher task name

Change a watcher task name from "Watch new files" to "WatchMyFolder" to be in sync with images.

eamonoreilly reviewed (approved):
Thanks Alexandair for the update to match the image.
